conda/recipes/ucxx/meta.yaml

```
@@ -127,9 +135,16 @@ outputs:
     requirements:
       build:
         - cmake
+      host:
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% else %}
+        - cuda-cudart-dev
+        {% endif %}
+        - cuda-version ={{ cuda_version }}
       run:
         - {{ pin_subpackage('libucxx', exact=True) }}
-        - cudatoolkit {{ cuda_spec }}
+        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```

To make sure the `libcudart` dependency is satisfied at `run` time, would suggest adding `cudatoolkit` here (especially since we are relaxing the version constraint on it above

```suggestion
        {% if cuda_major == "11" %}
        - cudatoolkit
        {% endif %}
        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```